crates/forge/src/scarb/config.rs

```diff
@@ -97,29 +95,25 @@ impl TryFrom<RawForgeConfig> for ForgeConfig {
         let mut fork_targets = vec![];
 
         for raw_fork_target in value.fork {
-            let block_id: Vec<BlockId> = raw_fork_target
-                .block_id
-                .iter()
-                .map(|(id_type, value)| match id_type.as_str() {
-                    "number" => BlockId::Number(value.parse().unwrap()),
-                    "hash" => BlockId::Hash(value.to_field_element()),
-                    "tag" => match value.as_str() {
-                        "Latest" => BlockId::Tag(BlockTag::Latest),
-                        _ => unreachable!(),
-                    },
-                    _ => unreachable!(),
-                })
-                .collect();
-
-            let [block_id] = block_id[..] else {
+            let [(block_id_type, block_id_value)] =
+                raw_fork_target.block_id.iter().collect_vec()[..]
+            else {
                 unreachable!()
             };
 
+            let block_id_type = match block_id_type.as_str() {
+                "number" => "Number",
+                "hash" => "Hash",
+                "tag" => "Tag",
+                _ => unreachable!(),
```

Comment:
May look weird, but I want the `unreachable!` arm https://en.wikipedia.org/wiki/Defensive_programming#Offensive_programming

Reply:
So, maybe we shouldn't handle it here?

Reply:
We decided to allow both lowercase and title case names